conans/test/functional/command/search_test.py

```diff
@@ -175,6 +177,24 @@ def setUp(self):
                                                PACKAGES_FOLDER,
                                                CONANINFO): conan_vars_tool_linx64},
                          self.client.cache.store)
+        # Fake metadata
+
+        def create_metadata(folder, pids):
+            metadata = PackageMetadata()
+            metadata.recipe.revision = DEFAULT_REVISION_V1
+            for pid in pids:
+                metadata.packages[pid].revision = DEFAULT_REVISION_V1
+            save(os.path.join(self.client.cache.store, folder, "metadata.json"), metadata.dumps())
+
+        create_metadata(root_folder1, ["WindowsPackageSHA", "PlatformIndependantSHA",
+                                       "LinuxPackageSHA"])
+        create_metadata(root_folder11, ["WindowsPackageSHA"])
```

Comment:
These changes in this tests, makes me wonder if it is not going to break all users cache that won't have a metadata.json file in their packages yet.

Reply:
Yes, it will. That's why I wrote there is a pending migration needed in this PR.